<Critical TimeStamp="2015-04-01T23:55:20.294" File="Consumer" Line="593" Routine="Unknown" Host="crc-06" Process="TelCalPublisherEventNC-Consumer" Thread="Thread-2" Context="" SourceObject="TelCalPublisherEventNC-Consumer" StackId="" StackLevel="0"><![CDATA[__process_event - Took too long to handle an 'PhaseCalReducedEvent' event: 2.257478 seconds.]]></Critical>

<Critical TimeStamp="2015-04-02T00:07:34.277" File="Consumer" Line="593" Routine="Unknown" Host="crc-06" Process="TelCalPublisherEventNC-Consumer" Thread="Thread-2" Context="" SourceObject="TelCalPublisherEventNC-Consumer" StackId="" StackLevel="0"><![CDATA[__process_event - Took too long to handle an 'AtmosphereReducedEvent' event: 2.159875 seconds.]]></Critical>

<Critical TimeStamp="2015-04-02T02:00:00.874" File="string" Line="138" Routine="Unknown" Host="gas01" Process="CCLSimpleClient" Thread="MainThread" Context="" SourceObject="CCLSimpleClient" StackId="" StackLevel="0"><![CDATA[Main - No Primary Flux Calibrator up in the sky! Wait until a primary Flux Calibrator is up]]></Critical>

<Critical TimeStamp="2015-04-02T02:00:14.940" File="ScriptExecutor" Line="166" Routine="Unknown" Host="gas01" Process="None" Thread="omniORB--1298793584" Context="" SourceObject="None" StackId="" StackLevel="0"><![CDATA[setExecError - Error in the script execution: ACSErr.ErrorTrace(file='/alma/ACS-12.1/ACSSW/bin/SBExecProcess.py', lineNum=138, routine='<module>', host='gas01', process='5159', thread='MainThread', timeStamp=136472328149382460L, sourceObject='', errorType=10100L, errorCode=5L, severity=Error, shortDescription='General ScriptExecutor runtime error', data=[], previousError=[ACSErr.ErrorTrace(file='/alma/ACS-12.1/ACSSW/bin/SBExecProcess.py', lineNum=132, routine='<module>', host='gas01', process='5159', thread='MainThread', timeStamp=136472328149346010L, sourceObject='', errorType=7L, errorCode=0L, severity=Error, shortDescription='A Python exception', data=[ACSErr.NameValue(name='Real Description', value='Request to exit from the interpreter.'), ACSErr.NameValue(name='Traceback', value='Traceback (most recent call last):\n  File "/alma/ACS-12.1/ACSSW/bin/SBExecProcess.py", line 108, in <module>\n    result = executor.runSource(srcCode)\n  File "/alma/ACS-12.1/ACSSW/lib/python/site-packages/ScriptExec/Executor.py", line 92, in runSource\n    exec source in self._globals, self._locals\n  File "<string>", line 140, in <module>\nSystemExit\n')], previousError=[])])]]></Critical>
